HW2_Voskoboinikov/calculator.py

```diff
@@ -0,0 +1,35 @@
+def addition(a, b):
+	return(a + b)
```

Comment:
скобки у return не нужны
```suggestion
	return a + b
```